content/questions/actions/question-081.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 081"
+question: "How can you reuse a defined workflow in multiple repositories?"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization
+1. [ ] By copying the workflow file to each repository
+1. [x] By using workflow templates
+1. [ ] By creating a reusable action
+1. [ ] By defining the workflow in a central repository
```

Comment:
Having a `central` repository with common reusable workflows is a widely used pattern and used as often or even more often than workflow templates.